doc/source/catalog_generation.rst

```diff
@@ -0,0 +1,361 @@
+.. _catalog_generation:
+
+==================
+Catalog Generation
+==================
+
+Point (Aperture) Photometric Catalog Generation
+================================================
+
+1: Source Detection
+-------------------
+
+1.1: Important Clarifications
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+As described in the previous step, AstroDrizzle creates a single multi-filter detector-level drizzle-combined
+image for source identification and one or more detector/filter-level drizzle-combined images (depending on
+which filters were used in the dataset) for photometry. The same set of sources identified in the
+multi-filter detection image is used to measure photometry for each filter. We use method to maximize the
+signal across all available wavelengths at the source detection stage, thus providing photometry with the
```

Comment:
Missing work in parentheses...We use (this) method to...

Reply:
Issue resolved in commit [203ed51](https://github.com/spacetelescope/drizzlepac/pull/760/commits/203ed519b49749f7ad519df95710174ab98faffe)